sentry/src/main/java/io/sentry/MainEventProcessor.java

```diff
@@ -85,6 +87,36 @@
               event.getEventId());
     }
 
+    // read packages name from options (without @version+build)
+    final String appPackage = "io.sentry.samples.android";
+    final String appJniPackage = appPackage.replace(".", "_");
+
+    // other way would be to pass the packages name as inAppIncludes to sentry-native, but they dont
+    // expose this option, yet.
```

Comment:
ideally, we'd go for https://github.com/getsentry/sentry-native/issues/472 instead of this hacky solution.